.sbtopts

```diff
@@ -0,0 +1,4 @@
+-J-Xss4m
```

Comment:
Is there a way to define these only on the cats-effect compilation SBT task?

Reply:
I don't believe so. Those need to be set before sbt initialises anyway I guess.

Reply:
@dom-nguyen: Do we need to have these settings? I'm able to compile the project locally without them (I'm using [paulp/sbt-extras](https://github.com/paulp/sbt-extras) which provides more sensible defaults), and Travis also seems happy without them (see https://travis-ci.org/pureconfig/pureconfig/builds/585745699).

Reply:
Hmm yeah I got an failure earlier in Travis wrt out of Metaspace, but now it's fine. Removed the sbtopts now.

Didn't know about sbt-extras, thanks for that!

Reply:
FYI, Travis also uses sbt-extras 👍